src/Playlist.tsx

```diff
@@ -33,10 +33,19 @@ export const PlayList: React.FC = () => {
     return <div>Playlist Loading from {endpoint}</div>;
   }
 
+  if (itemClicked !== -1){
+    return <div>{data.items[itemClicked].name}</div>
+  }
+  const onClick = (options: {key: number}): void => {
```

Comment:
you should keep your code consistently styled. 

newlines in between class items